docs/usage/secrets_rotation.md

```diff
@@ -0,0 +1,19 @@
+# Secrets and rotation
+
+## List of secrets
+
+**Cloudprovider Secret**
+
+*Example*: https://github.com/gardener/gardener/blob/master/example/70-secret-provider.yaml
+
+*Usage*: Authenticate gardener and kubernetes components for infrastructure operations
+
+*Rotation*: Rotating the cloudprovider secret requires multiple steps:
+
+1. Either create the new secret in the garden cluster and add a secretBinding or edit the current secret.
+2. :warning: Wait until all Shoots using the secret are reconciled before you deactivate the old secret! Otherwise the shoots will no longer function.
+3. After you are sure all Shoots using the secret are reconciled you can go ahead and deactivate the old secret in your infrastructure provider.
+
+*Description*: Gardener uses the cloudprovider secret to interact with the infrastructure when setting up shoot networks or security groups via the [terraformer](https://github.com/gardener/terraformer). It is also used by the cloud-controller-manager of your Shoot for example to create VMs. Depending on the cloudprovider the format of the secret differs. Please consult the example above and respective infrastructure extension documentation for the concrete layout. 
```

Comment:
Can you amend the example about CCM? I'm not aware that it creates VMs (that's MCM's job).
https://kubernetes.io/docs/concepts/architecture/cloud-controller/

Reply:
Yes, thanks for the remark. I think I got it now: The MCM creates the VMs. The CCM then is only interacting with Node objects (Node controller). I adapted the description. PTAL